phalcon/debug.zep

```diff
@@ -560,7 +560,7 @@ class Debug
 	/**
 	 * Throws an exception when a notice or warning is raised
 	 */
-	public function onUncaughtLowSeverity(severity, message, file, line)
+	public function onUncaughtLowSeverity(severity, message, file, line, context)
```

Comment:
The `file`, `line` and `context` args are optional. See [set_error_handler](http://php.net/manual/en/function.set-error-handler.php).

Mb something like? : 
```zephir
public function onUncaughtLowSeverity(severity, message, string file = __FILE__, int line = __LINE__, array context = null)
{
    ...
}
```